src/main/resources/META-INF/plugin.xml

```diff
@@ -1,41 +1,16 @@
+<!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
 <idea-plugin>
```

Comment:
The plugin.xml is merged from settings in the gradle build. 
The dist version will include the description, version, dependencies, compatibility range.

Change notes are not set.